cmd/acr/purge_test.go

```diff
@@ -606,6 +637,30 @@ var (
 	invalidLastUpdateTime = "date"
 
 	OneTagResult = &acr.RepositoryTagsType{
+		Response: autorest.Response{
+			Response: &http.Response{
+				StatusCode: 200,
+			},
+		},
+		Registry:  &testLoginURL,
+		ImageName: &testRepo,
+		TagsAttributes: &[]acr.TagAttributesBase{
+			{
+				Name:                 &tagName,
+				LastUpdateTime:       &lastUpdateTime,
+				ChangeableAttributes: &acr.ChangeableAttributes{DeleteEnabled: &deleteEnabled},
+				Digest:               &digest,
+			},
+		},
+	}
+
+	OneTagResultWithNext = &acr.RepositoryTagsType{
+		Response: autorest.Response{
+			Response: &http.Response{
+				StatusCode: 200,
+				Header:     http.Header{linkHeader: {"/acr/v1/&testRepo/_tags?last=latest&n=3&orderby=timedesc"}},
```

Comment:
Should we mock the real link header format here? Like `</acr/v1/&testRepo/_tags?last=latest&n=3&orderby=timedesc>; rel="next"`

Besides, can we have a few more test cases to cover different possible queries? Such as `</acr/v1/library/ubuntu/_tags?last=123%26latest&n=100&orderby=timedesc>; rel="next"`, `</acr/v1/library/ubuntu/_tags?orderby=&n=100&last=123%26latest>; rel="next"`, etc.

(`%26` is `&`)

Reply:
Added tests for this